civihr_default_theme/scss/base/_variables.scss

```diff
@@ -118,7 +118,7 @@ $nav-tabs-justified-active-link-border-color: $light !default;
 
 $screen-sm-max:                   767px !default;
 $crm-menu-responsive-breakpoint:  1250px;
-$crm-onboarding-wizard-menu-responsive-breakpoint:  1300px;
+$crm-onboarding-wizard-menu-responsive-breakpoint:  1170px;
```

Comment:
why was this changed to 1170px ? Why not [$screen-lg](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L314) which is 1200px?

Reply:
The idea is not to switch to Mobile view for the Progress Bar, otherwise it is looking weird ,
![welcome_to_civihr___hr17-4_](https://user-images.githubusercontent.com/5058867/32320356-83699eec-bfe3-11e7-87ad-7311916f2739.png)

So tried to lower the breakpoint, as much as I can.